packages/contracts/test/AccessControlTest.js

```diff
@@ -306,14 +310,14 @@ contract('Access Control: Liquity functions with the caller restricted to Liquit
     })
 
     // fallback (payment)	
-    it("fallback(): reverts when called by an account that is not the Active Pool", async () => {
+    it("fallback(): reverts because cannot receive ETH", async () => {
       // Attempt call from alice
       try {
         const txAlice = await web3.eth.sendTransaction({ from: alice, to: defaultPool.address, value: 100 })
         
       } catch (err) {
         assert.include(err.message, "revert")
-        assert.include(err.message, "DefaultPool: Caller is not the ActivePool")
+        //assert.include(err.message, "DefaultPool: Caller is not the ActivePool")
```

Comment:
:thinking:

Reply:
The tx reverts without that msg because the DefaultPool contract cannot receive ETH anymore